lib/Smokeping.pm

```diff
@@ -3310,12 +3310,12 @@ DOC
            hierarchies => {
               _doc => <<DOC,
 Provide an alternative presentation hierarchy for your smokeping data. After setting up a hierarchy in this
-section. You can use it in each tagets parent property. A drop-down menu in the smokeping website lets
+section. You can use it in each target's parent property. A drop-down menu in the smokeping website lets
```

Comment:
I hope that an `'` isn't a problem, I'm pretty sure this is a correct fix otherwise